src/pages/seulbiKim/Login/LoginSb.js

```diff
@@ -1,7 +1,110 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './LoginSb.scss';
 
-function Login() {
-  return <div>Login</div>;
-}
+const LoginSb = () => {
+  const navigate = useNavigate();
+  const goToMain = () => {
+    navigate('/main-seulbiKim');
+  };
 
-export default Login;
+  let [input, setInput] = useState({
+    id: '',
+    pw: '',
+  });
+  const { id, pw } = input;
+  const isInputValid = id.includes('@') && pw.length >= 5 ? false : true;
+
+  const onChangeInput = e => {
+    e.preventDefault();
+    const { name, value } = e.target;
+    setInput({
+      ...input,
+      [name]: value,
+    });
+  };
+
+  // 로그인 통신
+  const loginPost = e => {
+    e.preventDefault();
+    fetch('http://10.58.4.15:8000/users/login', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        id: id,
+        pw: pw,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => {
+        if (result.access_token) {
+          localStorage.setItem('token', result.access_token);
+          goToMain();
+        }
+      });
+  };
+  let token = localStorage.getItem('token') || '';
+  console.log(token);
+
+  // 회원가입 통신
+  // const signUpPost = e => {
+  //   e.preventDefault();
+  //   fetch('http://10.58.4.15:8000/users/signup', {
+  //     method: 'POST',
+  //     body: JSON.stringify({
+  //       id: id,
+  //       pw: pw,
+  //       name: 'seul',
+  //       phonenumber: '01012345678',
+  //       personal: 'test',
+  //     }),
+  //   })
+  //     .then(response => response.json())
+  //     .then(result => console.log('결과: ', result));
+  // };
+
+  return (
+    <div className="loginPage">
```

Comment:
```suggestion
    <div className="loginSB">
```
scss를 사용할 때는 최상단 부모 태그의 className은 컴포넌트 이름과 맞춰주는 것이 좋습니다